docs/howto/manage-storage.md

```diff
@@ -164,7 +96,107 @@ The storage will not be available immediately after that call - the charm should
 observe the `storage-attached` event and handle any remaining setup once Juju
```

Comment:
I can't attach this to the right line because it hasn't changed, but I believe there should be a "range" for the "scratch" storage, like the one above it. "multiple: range: 1-10" rather than "multiple: 1-10". I would probably again remove the "location" field as well.

Just above that, there's also a "storage-attached" that should probably be "`storage-attached`".